gallery/src/components/NxTransferList/NxTransferListPage.tsx

```diff
@@ -59,13 +61,31 @@ const NxTransferListPage = () =>
             </NxTable.Row>
             <NxTable.Row>
               <NxTable.Cell><NxCode>selectedItems</NxCode></NxTable.Cell>
-              <NxTable.Cell>Set</NxTable.Cell>
+              <NxTable.Cell>Set | Array</NxTable.Cell>
               <NxTable.Cell>Yes</NxTable.Cell>
               <NxTable.Cell></NxTable.Cell>
               <NxTable.Cell>
-                A Set containing the ids of the items which are selected – that is, the ones which should appear
-                on the right side of the component. Be sure to always provide a fresh <NxCode>Set</NxCode> instance
-                when updating this prop, rather than mutating the same <NxCode>Set</NxCode>.
+                A Set, or an Array if <NxCode>allowReordering</NxCode> is set to true, containing the ids of the
+                items which are selected – that is, the ones which should
+                appear on the right side of the component. Be sure to always provide a fresh
+                <NxCode>Set</NxCode> or <NxCode>Array</NxCode> instance
```

Comment:
Fixes whitespace around the `<NxCode>`
```suggestion
                appear on the right side of the component. Be sure to always provide a
                fresh <NxCode>Set</NxCode> or <NxCode>Array</NxCode> instance
```